qiling/os/windows/dlls/ntoskrnl.py

```diff
@@ -899,14 +902,17 @@ def hook_IoCreateDriver(ql: Qiling, address: int, params):
     init_sp = ql.os.init_sp
 
     ql.os.fcall = ql.os.fcall_select(STDCALL)
-    ql.os.fcall.writeParams(((POINTER, ql.driver_object_address), (POINTER, ql.regitry_path_address)))
+    ql.os.fcall.writeParams((
+        (POINTER, ql.driver_object_address),
```

Comment:
maybe we should move this to ql.os and not expose it to the core ?

this must be some old code i over looked

Reply:
Move which part exactly..?
This was a manual function call code I replaced with the new-style fcall. Didn't really got into why it is there at the first place.

We can fix that in another PR.